client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -70,16 +73,17 @@ class DomainDetailsForm extends PureComponent {
 			isDialogVisible: false,
 			submissionCount: 0,
 			phoneCountryCode: 'US',
-			registrantExtraInfo: null,
 			steps,
 			currentStep: first( steps ),
 		};
 	}
 
+
 	componentWillMount() {
 		this.formStateController = formState.Controller( {
 			fieldNames: this.fieldNames,
-			loadFunction: wpcom.getDomainContactInformation.bind( wpcom ),
+			initialFields: this.props.contactDetails,
+			// loadFunction: this.loadFormState,
```

Comment:
I don't think we can actually rely on `initialFields` init because of the way `formStateController` currently works. It won't null init the object properties that are null (eg, `fax`, `address2`) as-is. So we either need to edit the way the controller works in init or do what it had been doing before w/fields + `loadFunction`.